docs/index.rst

```diff
@@ -83,7 +83,13 @@ Permissions on vCenter
 OS Templates
 ~~~~~~~~~~~~
 
-* You need two OS templates of your preferred operating systems (e.g. Ubuntu Trusty) within the vSphere datastores. One for the Cloudify manager and one for the application VMs. The application VM template should accept the Cloudify agent public key for its root user. The Cloudify manager template must accept the cloudify manager public key. Note that you can choose to use same template for both the manager and the application VMs, in that case the shared template must accept both public keys.
+* You need two OS templates within the vSphere datastores.
+  One for the Cloudify manager and one for the application VMs.
+  The Cloudify manager template must have CentOS 7 installed.
+  The application VM template should accept the Cloudify agent public key for its root user.
+  The Cloudify manager template must accept the cloudify manager public key.
+  Note that you can choose to use same template for both the manager and the application VMs,
```

Comment:
`Note that you can choose to use same template for both ~the manager~ *Cloudify Manager* and the application VMs`...